yktrace/src/hwt/mapper.rs

```diff
@@ -94,17 +94,18 @@ fn extract_labels() -> Result<Vec<(u64, (String, u32))>, gimli::Error> {
     // Extract labels
     let mut labels = Vec::new();
 
-    let loader = |id: gimli::SectionId| -> Result<borrow::Cow<[u8]>, gimli::Error> {
+    let loader = |id: gimli::SectionId| -> Result<&[u8], gimli::Error> {
         Ok(object
-            .section_data_by_name(id.name())
-            .unwrap_or(borrow::Cow::Borrowed(&[][..])))
+            .section_by_name(id.name())
+            .map(|sec| sec.data().expect("failed to decompress section"))
```

Comment:
What impact does compression have on performance?

Reply:
I don't know. ykrustc is not emitting compressed elf sections. They are an ELF extension that is as far as I know only used for debug info in some cases.